src/openfermion/utils/_trotter_exp_to_qgates_test.py

```diff
@@ -0,0 +1,172 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Tests for _trotter_exp_to_qgates.py"""
+
+import unittest
+import os
+import cStringIO
```

Comment:
cStringIO is not python3 compatible.  In Python3 one just uses stringio and it attempts to find the fastest implemented version.